torchmetrics/functional/audio/pesq.py

```diff
@@ -98,3 +102,23 @@ def pesq(preds: Tensor, target: Tensor, fs: int, mode: str, keep_same_device: bo
         pesq_val = pesq_val.to(preds.device)
 
     return pesq_val
+
+
+@deprecated(target=perceptual_evaluation_speech_quality, deprecated_in="0.7", remove_in="0.8", stream=_future_warning)
+def pesq(preds: Tensor, target: Tensor, fs: int, mode: str, keep_same_device: bool = False) -> Tensor:
```

Comment:
are there any deprecation tests?

Reply:
we are still in a pre-stable (v1.0) so we do not check the correct raised warning as I believe it works fine as we use it all other PL family packages :]